lib/pgbackups.js

```diff
@@ -64,7 +64,10 @@ module.exports = (context, heroku) => ({
       }
     }
   },
-  wait: (action, transferID, interval, verbose) => {
+  wait: (action, transferID, interval, verbose, app) => {
```

Comment:
I admit this is pretty gross. I welcome suggestions for a better approach, but this does seem to work.